adaptors/hybrisalsadaptor/hybrisalsadaptor.cpp

```diff
@@ -55,7 +55,7 @@ void HybrisAlsAdaptor::stopSensor()
 void HybrisAlsAdaptor::processSample(const sensors_event_t& data)
 {
     TimedUnsigned *d = buffer->nextSlot();
-    d->timestamp_ = quint64(data.timestamp * .0001);
+    d->timestamp_ = quint64(data.timestamp * .001);
```

Comment:
Well, it would apply to all of them. `data.timestamp` being 0 doesn't matter - division _of_ zero is just zero.